### ISSUE TITLE
[Intro to Pytorch], "logits" output name causes confusing?

### ISSUE THREAD

**Issue by hpkhanh1610** [udacity/deep-learning-v2-pytorch#11]
Hi,

For the notebook "Part 3 - Training Neural Networks (Solution).ipynb" in Intro to Pytorch, in the 3rd code cell (I pasted it below), I see that we use the nn.LogSoftmax() layer at the end, so I think at the forward pass step, **logits = model(images)**, the name of the output is not **logits**, I think we should use another name, like just "output" or "logsoftmax_output". 

```# Build a feed-forward network
model = nn.Sequential(nn.Linear(784, 128),
                      nn.ReLU(),
                      nn.Linear(128, 64),
                      nn.ReLU(),
                      nn.Linear(64, 10),
                      **nn.LogSoftmax(dim=1)**)

# Define the loss
criterion = nn.NLLLoss()

# Get our data
images, labels = next(iter(trainloader))
# Flatten images
images = images.view(images.shape[0], -1)

**# Forward pass, get our logits
logits = model(images)
# Calculate the loss with the logits and the labels
loss = criterion(logits, labels) ```**

**Comment by mcleonard** [comment]
Right, I'll fix this.

**Comment by mcleonard** [comment]
Fixed in this commit: 05105e722dfa521aa1501bea3426bcc8fadfd203
